pages/faq.md

```diff
@@ -1,57 +1,47 @@
+Een videotuul is een audiovisueel verslag (samenvoeging van video en notulen) van zo'n vergadering. Als het alleen een geluidsopname is wordt het een audiotuul genoemd. Naast de opname bestaat een videotuul uit metagegevens die contextinformatie geven over de vergadering, zoals agendapunten, sprekersinformatie, spreekfragmenten en aanwezige deelnemers. Domeinspecifieke gegevens dus.
 
-# Hoe verhoudt ORI-A zich tot ORI?
+# Moeten videotulen worden bewaard?
+Strikt genomen zijn overheden niet altijd verplicht om videotulen te bewaren. Als er ook schriftelijke verslagen zijn van vergaderingen, dan hoort de meest informatierijke bron te worden bewaard. Vanuit cultuurhistorisch oogpunt is het wel belangrijk dat videotulen permanent bewaard worden. Videotulen zijn een belangrijk onderdeel van de digitale democratie.
 
-ORI (Open Raadsinformatie, ook wel [Open Raads-, Staten- en Bestuursinformatie](https://vng-realisatie.github.io/ODS-Open-Raadsinformatie/)) is een verzamelnaam voor een aantal initiatieven die met elkaar samenhangen rondom het thema van actieve openbaarmaking van informatie uit het politiek en bestuurlijk besluitvormingsproces. 
+# Waarom is ORI-A ontwikkeld?
+ORI-A is ontwikkeld als domeinspecifieke standaard voor raadsinformatie, wanneer dit voor permanente bewaring naar een [e-depot](https://www.nationaalarchief.nl/archiveren/kennisbank/wat-is-een-e-depot) wordt gemigreerd. Om videotulen op een [duurzaam toegankelijke] (https://www.nationaalarchief.nl/archiveren/kennisbank/duurzaam-toegankelijk) manier te kunnen beheren en beschikbaar stellen. Om migratie van raadsinformatie uniformer en efficiënter te maken, en zeker te zijn dat de juiste metagegevens meekomen.
 
-Vanaf 2015 heeft een samenwerking tussen de Open State Foundation, VNG Realisatie, Argu en Ontola het platform openraadsinformatie.nl (nu: openbesluitvorming.nl) opgeleverd. Dit platform richtte zich op het ophalen en vervolgens doorzoekbaar maken van zoveel mogelijk digitaal gepubliceerde politieke en bestuurlijke besluitvormingsstukken van gemeenten en dit op herbruikbare wijze als open data te ontsluiten. Sinds de intrede van de Wet open overheid in 2022 is de scope uitgebreid naar alle Nederlandse politieke en bestuurlijke overheidsorganen. 
+# Waarom is ORI-A een XML-schema?
+ORI-A is als XML Schema Document (XSD) ontworpen, zodat het gezamenlijk met [MDTO](https://www.nationaalarchief.nl/archiveren/mdto) kan worden gebruikt bij migraties naar het e-depot. Metagegevens over informatieobjecten en bestanden kan dan in MDTO worden uitgedrukt, en metagegevens over raadsinformatie in ORI-A.
```

Comment:
~~Metagegevens over informatieobjecten en bestanden kan dan in MDTO worden uitgedrukt~~ _kunnen_ dan in MDTO worden uitgedrukt